### PR TITLE
added a service worker that caches index.html and main.js

### PR DESCRIPTION
**Pull request by billy-hardy**

### CODE REVIEW

File: install.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+if('serviceWorker' in navigator) {
+	navigator.serviceWorker.register('/sw.js').then(function(registration) {
+		// Registration was successful
```
Comment (Red-Thirteen) [inline install.js]: isn't the comment redundant 🎭

File: install.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+		console.log('ServiceWorker registration successful with scope: ',
+			registration.scope);
+	}).catch(function(err) {
+		// registration failed :(
```
Comment (Red-Thirteen) [inline install.js]: I'll allow it since the sad face is there

File: install.js (hunk 1 above)
Comment (billy-hardy) [inline install.js]: fixed it

**Comment by GustavoFernandes** [comment]
This causes bug #12, this pull request can be reopened when resolved

**Pull request merged**